app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -31,6 +37,19 @@ def datacite_endpoint
 
     private
 
+      def validate_email_format
+        return unless settings['email_format'].present?
+        settings['email_format'] = settings['email_format'].grep(/@\S*\.\S*/)
+      end
+
+      def validate_contact_emails
+        email_validation_format = URI::MailTo::EMAIL_REGEXP
+        ['weekly_email_list', 'monthly_email_list', 'yearly_email_list'].each do |key|
```

Comment:
Should we add a class property + accessor for the weekly, monthly and yearly list array?

@@AVAILABLE_EMAIL_PERIODS = %w(week month year).map{ |period| "#{period}ly_email_list" }
cattr_reader :AVAILABLE_EMAIL_PERIODS

We could even set the periods on the settings.yml file